src/modules/Landing/components/Footer/Footer.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import '../styles/Footer.css'
+import '../../styles/Footer.css'
```

Comment:
Although you can put all your .css files in one /styles folder, it is more appropriate to have the .css file with the component such as:

/components/footer/index.js.  => component file
/components/footer/footer.css => css for footer component